templates/eap-cd-image-stream.json

```diff
@@ -0,0 +1,106 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss-cd-image-streams",
+        "annotations": {
+            "description": "ImageStream definitions for JBoss Enterprise Application Platform Continuous Delivery.",
+            "openshift.io/provider-display-name": "Red Hat, Inc."
+        }
+    },
+    "items": [
+    {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "eap-cd-openshift",
+                "annotations": {
+                    "openshift.io/display-name": "JBoss EAP Continuous Delivery",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "version": "12.0"
+                }
+            },
+            "labels": {
+                "xpaas": "1.4.10"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "latest",
```

Comment:
@rcernich WDYT about this use of latest? We'd have "latest" and "12" as tags in here, with "latest" always pointing to the most recent.